02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,78 @@
+body{
+  background-color: #FAEBD7;
+}
+
+
+  header{
+text-align: center;
+}
+h1{
+  color: blue
+  font-family: "Times New Roman", Times, serif;
+}
+
+
+ol{
+  font-family: "Times New Roman", Times, serif;
+}
+
+p{
+  font-family: "Times New Roman", Times, serif;
+}
+
+section{
+  float: left;
+}
+
+aside{
+  float: right;
+  padding: 30px;
+}
+a{
+  text-decoration: none;
+}
+h2{
+  padding-bottom: 20px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+ul li{
+  margin-left: 20px;
+  padding-top: 20px;
+  padding-left: 10px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+table{
+  float: left;
+  margin-left:auto;
+  margin-right: auto;
+}
+
+th{
+  padding: 20px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+p.table{
```

Comment:
This is a great effort. Clearly you designed something great here. Question: What is this dot notation supposed to do?